network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/responsive_screen_wide_image.html

```diff
@@ -0,0 +1,60 @@
+{% load wagtailimages_tags %}
+
+<picture
+>
+    {% comment %}
+    Screen sizes
+    small: "576px",
+    medium: "768px",
+    large: "992px",
+    xlarge: "1200px",
+    "2xl": "1400px",
+  {% endcomment %}
+    {% if not is_pattern_library %}
+        {% image image_object width-576 as img_576 %}
+        {% image image_object width-1152 as img_576_2x %}
+        {% image image_object width-768 as img_768 %}
+        {% image image_object width-1536 as img_768_2x %}
+        {% image image_object width-992 as img_992 %}
+        {% image image_object width-1984 as img_992_2x %}
+        {% image image_object width-1200 as img_1200 %}
+        {% image image_object width-2400 as img_1200_2x %}
+        {% image image_object width-1400 as img_1400 %}
+        {% image image_object width-2800 as img_1400_2x %}
+        {% image image_object original as img_original %}
+    {% endif %}
+    <source
+        media="(max-width: 576px)"
+        srcset="{{ img_576.url }}, {{ img_576_2x.url }} 2x"
+    />
+    <source
+        media="(max-width: 768px)"
+        srcset="{{ img_768.url }}, {{ img_768_2x.url }} 2x"
+    />
+    <source
+        media="(max-width: 992px)"
+        srcset="{{ img_992.url }}, {{ img_992_2x.url }} 2x"
+    />
+    <source
+        media="(max-width: 1200px)"
+        srcset="{{ img_1200.url }}, {{ img_1200_2x.url }} 2x"
+    />
+    <source
+        media="(max-width: 1400px)"
+        srcset="{{ img_1400.url }}, {{ img_1400_2x.url }} 2x"
+    />
+    <source
+        media="(min-width: 1401px)"
+        srcset="{{ img_original.url }}"
+    />
+    <img
+        class="tw-h-full tw-w-full tw-object-cover"
+        {% comment %}
+      This is only loaded when the browser does not know the picture and source element.
+      Trying to fall back to some reasonably sized image that should work on most screens.
+      Assuming low probability for a high res screen (since it's an outdated browser).
+    {% endcomment %}
+        src="{{ img_1200.url }}"
+        alt=""
```

Comment:
Might be good to include the alt text here in case the hero isn't always decorative.

Reply:
Great idea @robdivincenzo! This has been updated in the latest commit, please see [this comment](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12122/files#r1550306674) for more details.